README.md

```diff
@@ -170,7 +170,7 @@ To run Bats' internal test suite (which is in the container image at
 To run a test suite from your local machine, mount in a volume and direct Bats
 to its path inside the container:
 
-    $ docker run -it -v "$(pwd):/code" bats/bats:latest /code/test
+    $ docker run -it -v "$(pwd):/code" bats/bats:latest /test
```

Comment:
Should this be `./test` instead of `/code/test`?

Reply:
Oh, yeah! Actually just `test` would make the trick, but `/test` is wrong indeed.